airflow/requirements.txt

```diff
@@ -15,3 +15,7 @@ lxml==4.9.3
 tiktoken==0.5.1
 firebase-admin==6.2.0
 snowflake-connector-python==3.6.0
+tenacity==8.2.3
+html2text==2020.1.16
+
+astro-run-dag # This package is needed for the astro run command. It will be removed before a deploy
```

Comment:
why is astro run command needed?

Reply:
I think this gets auto added by the `astro` command from asto sdk. Not sure why it didn't get auto removed this time. Let me get rid of it. Nice catch!